src/components/globalConfigurations/GlobalConfiguration.tsx

```diff
@@ -390,69 +448,82 @@ function Body({ getHostURLConfig, checkList, serverMode, handleChecklistUpdate }
+            {!window._env_.K8S_CLIENT && (
+                <Route
+                    path={`${URLS.GLOBAL_CONFIG_DOCKER}/:id?`}
+                    render={(props) => {
+                        return (
+                            <div className="flexbox">
+                                <Docker {...props} handleChecklistUpdate={handleChecklistUpdate} />
+                            </div>
+                        )
+                    }}
+                />
+            )}
 
-            <Route
-                path={URLS.GLOBAL_CONFIG_CHART}
-                render={(props) => {
-                    return <ChartRepo />
-                }}
-            />
-            <Route path={URLS.GLOBAL_CONFIG_CUSTOM_CHARTS}>
-                <CustomChartList />
-            </Route>
-            <Route
-                path={URLS.GLOBAL_CONFIG_LOGIN}
-                render={(props) => {
-                    return <SSOLogin {...props} />
-                }}
-            />
-            <Route
-                path={URLS.GLOBAL_CONFIG_AUTH}
-                render={(props) => {
-                    return <UserGroup />
-                }}
-            />
-            <Route
-                path={`${URLS.GLOBAL_CONFIG_NOTIFIER}/edit`}
-                render={(props) => {
-                    return <AddNotification {...props} />
-                }}
-            />
-            <Route
-                path={URLS.GLOBAL_CONFIG_NOTIFIER}
-                render={(props) => {
-                    return <Notifier {...props} />
-                }}
-            />
-            <Route path={URLS.GLOBAL_CONFIG_EXTERNAL_LINKS}>
-                <ExternalLinks />
-            </Route>
-            <Redirect
-                to={
-                    serverMode &&
-                    (serverMode === SERVER_MODE.EA_ONLY ? URLS.GLOBAL_CONFIG_PROJECT : URLS.GLOBAL_CONFIG_HOST_URL)
-                }
-            />
+            {!window._env_.K8S_CLIENT && (
+                <Route
+                    path={URLS.GLOBAL_CONFIG_CHART}
+                    render={(props) => {
+                        return <ChartRepo />
+                    }}
+                />
+            )}
+            {!window._env_.K8S_CLIENT && (
+                <Route path={URLS.GLOBAL_CONFIG_CUSTOM_CHARTS}>
+                    <CustomChartList />
+                </Route>
+            )}
+            {!window._env_.K8S_CLIENT && (
+                <Route
+                    path={URLS.GLOBAL_CONFIG_LOGIN}
+                    render={(props) => {
+                        return <SSOLogin {...props} />
+                    }}
+                />
+            )}
+            {!window._env_.K8S_CLIENT && (
+                <Route
+                    path={URLS.GLOBAL_CONFIG_AUTH}
+                    render={(props) => {
+                        return <UserGroup />
+                    }}
+                />
+            )}
+            {!window._env_.K8S_CLIENT && (
```

Comment:
Same for all these routes. Create a set/list and render based on `window._env_.K8S_CLIENT` condition

Reply:
done